### PR TITLE
docs: update to latest typedoc and Typescript 4.2

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: documentation/typedoc.json [hunk 1]
```diff
@@ -1,15 +1,13 @@
 {
   "excludeExternals": true,
-  "externalPattern": ["../viewer/src/__tests__/**"],
+  "externalPattern": ["../viewer/src/__tests__/**", "../viewer/node_modules/**"],
   "exclude": ["**/*+(.spec|.test).ts"],
   "excludePrivate": true,
   "excludeProtected": true,
-  "stripInternal": true,
+  "excludeInternal": true,
```
Comment (christjt) [inline documentation/typedoc.json:7]: Is there a semantic difference here, or simply a rename?

Comment (maksnester) [reply documentation/typedoc.json:7]: yeah, reference to a doc or just common text in the PR description would be great to quickly understand the reasons behind changes 🙂

Comment (larsmoa) [reply documentation/typedoc.json:7]: Sorry, it's removed and picked up from `tsconfig.json` but I have it here to be "sure"... See https://github.com/TypeStrong/typedoc/issues/1469

**Review by maksnester — APPROVED** [review]
LGTM

**Pull request merged**